templates/statefulset.yaml

```diff
@@ -81,21 +81,49 @@ spec:
           - name: DEST_TRANSFORM_URL
             value: "http://{{ include "transformer.fullname" . }}:{{ .Values.transformer.service.port}}"
           - name: COMPUTE_DB_HOST_IN_K8S
+            {{- if .Values.external_postgresql.enabled }}
```

Comment:
Having two boolean fields viz. external_postgresql.enabled and postgresql.enabled can cause consistency issues. Does it make sense to use just postgresql.enabled?